client/me/purchases/billing-history/vat-vendor-details.tsx

```diff
@@ -106,10 +106,7 @@ export function VatVendorDetails( { transaction }: { transaction: BillingTransac
 			<strong>{ translate( 'Vendor VAT Details' ) }</strong>
 			<span>
 				{ vendorInfo.address.map( ( addressLine ) => (
-					<>
-						{ addressLine }
-						<br />
-					</>
+					<div key={ addressLine }>{ addressLine }</div>
```

Comment:
When mapping through an array to render a list, adding a key in React is recommended to identify the changed components. You have done that by providing a unique key, and now the earlier console error is not appearing anymore.